packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -269,7 +274,7 @@ export class Users {
       userStakedRep,
       userPositions,
       userPositionTotals,
-      marketsInfo
+      marketsInfo: [...marketList.markets, ...marketsInfo]
```

Comment:
`marketsInfo: [...marketList.markets, ...marketsInfo]` I believe this would be better if you invert the values, ilke this: `marketsInfo: [...marketsInfo, ...marketList.markets]`, no?

Otherwise the values from marketList might be overwritten by marketsInfo

Reply:
overridden is fine. I didn't want to take the performance hit to pull out marketIds and re-get them.